homepairs/homepairsUI/src/state/account/reducer.tsx

```diff
@@ -1,25 +1,37 @@
+/**
+* A reducer is a pure function that takes the previous state and 
+* an action as arguments and returns a new state. The reducer is 
+* instrumental in keeping the current state updated 
+* throughout our app as it changes. 
+* */
+
 import { 
     AccountState, 
     FetchUserAccountProfileAction, 
     AccountStateAction, 
 } from '../types';
 import { FETCH_PROFILE_ACTION_TYPES } from './actions';
 
-/**
-* A reducer is a pure function that takes the previous state and 
-* an action as arguments and returns a new state. The reducer is 
-* instrumental in keeping the current state updated 
-* throughout our app as it changes. 
-* */
 
 export const initialState: AccountState = null;
 
+/**
+ * ----------------------------------------------------
+ * accountProfile
+ * ----------------------------------------------------
+ * The reducer for the redux-store that deals with the state of anything homepairs
+ * account related. This reducer will be initialized upon authorization and will 
+ * be called during a change of user settings. Only dispatch methods and the 
+ * root store should ever need to use this function.
+ * @param {AccountState} state - current state of the redux-store
+ * @param {AccountStateAction} action - the value that determines what type of mutation will occur with the store 
+ */
 export const accountProfile = (
     state: AccountState = initialState,
     action: AccountStateAction,
 ) => {
     let profileState: AccountState;
-    const prevState = {...state, modalOpen: true };
```

Comment:
Can't verify if this is a good change or not. Might want to ask Cesar

Reply:
modalOpen does nothing to our components. It should not be included in the state. Our components individually take care of their own modals. Any time a modal is affected outside of the component, it is via callback. Never the state.

Reply:
our modals are decoupled from our reducer, so their state shouldn't be tracked in the reducer anymore

Reply:
yeah modalOpen does not affect my modals. Should be fine.